### ISSUE TITLE
Removing unwanted migration from the app

### ISSUE THREAD

**Issue by isuriupeksha** [systers/language-translation#28]
```ruby 
class RemoveUsernameFromUsers < ActiveRecord::Migration
  def change
    remove_column :users, :username, :string
  end
end
```
https://github.com/systers/language-translation/blob/master/PLTcode/db/migrate/20140626003527_remove_username_from_users.rb

@SaumyaGurtu I found this migration to remove `username` from the `users`  table. But the problem is that, `username` attribute haven't created by previous migration files. 

So we can remove that.

**Comment by SaumyaGurtu** [comment]
Please check if devise has already created username.

**Comment by isuriupeksha** [comment]
@SaumyaGurtu  Not. `username` haven't created by devise first. 

Therefore we can remove following migration as well 
`20140625230123_remove_email_from_users.rb`
`20140625230914_add_email_to_users.rb`

Email already in devise migration.

**Comment by wonook** [comment]
Be sure to drop & create & migrate a separate test db and try it before sending a pull request for this. It will be a big problem if things get tangled up with the db.

**Comment by isuriupeksha** [comment]
@wonook 
By removing above migration, you will not get any issue if you setup database again. If you carefully check this migration (https://github.com/systers/language-translation/blob/master/PLTcode/db/migrate/20140626003527_remove_username_from_users.rb) you may understand what is the issue.

**Comment by wonook** [comment]
Yeah, I've just checked, and some of them I wonder how it got in there in the first time. I'll quickly send a PR regarding this issue as there aren't any branches yet handling this issue.

**Comment by wonook** [comment]
by the way, I'm also curious about the roles. Is there any reason for converting the roles to a integer than as string which is default?

**Comment by wonook** [comment]
Check the PR and provide a quick feedback please!

**Comment by isuriupeksha** [comment]
@wonook great. I remember when I run the migration, I got that `username` error. But I keep it way from the migration list and setup the application.

**Comment by isuriupeksha** [comment]
@wonook Regarding the `role` --> `role_id` and why I convert to integer

---------

Concept: *Relational Data*

Since we have used relational models, we know that how relational database structures work.

Essentially, it does everything with id's (called a primary_key). This allows different models to call data from related models by calling the respective ID's. That's why I had to add role_id to your user model. 

But creating a `Role` table is not necessary, Because `Role` object should cached in the application without using ActiveModel. So `Role` is a simple class and has ability to act as a User's role object. 

I am trying to use the `delegate` method to load role data object into user model. This would have to work like this, `@user.admin?`
